frontend/app/components/context-menus/wp-context-menu/wp-context-menu.controller.ts

```diff
@@ -117,12 +117,6 @@ function wpContextMenuController($scope:any,
       $window.location.href = link;
       return;
     }
-
-    var params = {
-      workPackageId: selected[0].id
-    };
-
-    $state.transitionTo('work-packages.show.edit', params);
```

Comment:
I agree, and have changed the bulk actions to be explicit (Bulk move, edit, copy, delete).